code/game/machinery/doors/airlock.dm

```diff
@@ -646,11 +646,6 @@ About the new airlock wires panel:
 					return
 			else /*if(src.justzap)*/
 				return
-		else if(user.hallucination > 50 && prob(10) && src.operating == 0)
```

Comment:
From what I've obtained, this should be kept.

And I kind of agree. It's funny.